assets/js/components/ReportError.js

```diff
@@ -67,7 +70,28 @@ export default function ReportError( { moduleSlug, error } ) {
 		purify.sanitize( message, { ALLOWED_TAGS: [] } )
 	);
 
-	return <CTA title={ title } description={ description } error />;
+	const retry = !! error?.selectorData?.storeName;
```

Comment:
This variable name could be a bit more descriptive, and maybe check for all the arguments required to invalidate a selector, eg:

```suggestion
	const showRetry = error?.selectorData?.storeName && error?.selectorData?.storeName && error?.selectorData?.name && selectorData?.args;
```

Reply:
I think checking for the `storeName` and the `name === 'getReport'` should be sufficient like so:
```js
const showRetry = !! (
	error?.selectorData?.storeName &&
	error.selectorData?.name === 'getReport'
);
```